R/read_hivproj_param.R

```diff
@@ -128,6 +157,30 @@ get_art_mortality <- function(dp_data) {
   art_mort
 }
 
+#' Interpret CD4 mortality scale from version number.
+#'
+#' @param valid_version The version number.
+#'
+#' @keywords internal
+get_scale_cd4_mortality <- function(valid_version) {
+  version <- valid_version
+  beta_version <- NULL
+
+  if (is.character(valid_version)) {
```

Comment:
Is this version check used elsewhere? It looks like it's not specific to this function

Reply:
This is the only example (so far) that uses a version instead of looking at the tags. I think because this information is not persisted in the file itself.